FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/JobConfigurationConstants.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Jobs
+{
+    public static class JobConfigurationConstants
+    {
+        /// <summary>
+        /// The number of patients in each task, used in group filter scope.
+        /// </summary>
+        public const int NumberOfPatientsPerTask = 100;
+
+        /// <summary>
+        /// Time interval to sync job to store.
+        /// </summary>
+        public const int UploadDataIntervalInSeconds = 30;
+
+        /// <summary>
+        /// The cache resources number, if there are more resources than this value in cache, will commit it to storage.
+        /// </summary>
+        public const int NumberOfResourcesPerCommit = 10000;
```

Comment:
Guess this constant is used to restrict the total data ***count*** in a batch, do we have related constraints for the total data ***size*** in a batch?

Reply:
We don't have constraints for batch data size now, will add it later if necessary.

Reply:
now, we set the search bundle size to 10M to limit the cache data size. With this setting, the maximum memory usage is 3GB in our performance test.